### PR TITLE
Work samples

### PR DESCRIPTION
**Pull request by mirisuzanne**
Baby donkey with ladybug shirt

## Links:
- [Trello card](https://trello.com/c/GkPYooda/39-what-is-the-design-for-the-work-list-page)
- [Preview](https://deploy-preview-45--oddleventy.netlify.com/)

## Description
- [x] Description is in Trello card

## Impacted areas of website
- "Work" page 
- Blockquotes across the site should still look the same, but use a more general pattern

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[What is the design for the Work List page?](https://trello.com/c/GkPYooda/39-what-is-the-design-for-the-work-list-page)

File: content/work/cfoshare.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
 ---
 title: CFO Share Website
-sub: Built with WordPress
+sub: WordPress CMS & responsive design
 logo: cfoshare
-client: CFO Share
+client: &client CFO Share
```
[the remark this answers is not in the document]
Comment (mirisuzanne) [reply content/work/cfoshare.md]: See `*client` on line 14 – it's a way of making internal references in YAML.

**Comment by mirisuzanne** [comment]
> I know I've asked this before, but I don't recall a clear answer. Do we have explicit permission to use all these photos and logos from past clients? 

We don't. 

> Some of them look like outdated head shots we just found on the web -- 

They are mostly current LinkedIn profile pics at this point…

> I'd prefer we have clear written permission. 

I agree, especially for logos. We also need a fallback for bragging about clients when we can't use the logo. I think that's a new story we should add to Trello, and should not hold up this branch. 

> Same for quotes, though I'm guessing we already have that.

Yes, all the quotes were provided specifically for this (or are quoted from public twitter threads in the case of talk feedback).

**Comment by SondraE** [comment]
@mirisuzanne I noticed that some quote have bold text and some don't. Is that intentional or did we just forget to go through and find important parts in each? That's nothing that should hold up this branch, though.

**Comment by mirisuzanne** [comment]
@SondraE Not intentional. I didn't know we were supposed to have bold in all quotes, but you're welcome to make a story for it if that's the goal.

**Pull request merged**